apps/api/src/workspace-role/service/workspace-role.service.ts

```diff
@@ -94,6 +98,45 @@ export class WorkspaceRoleService {
             }
           }
         },
+        select: {
+          id: true
+        }
+      })
+    )
+
+    if (dto.projectEnvironments) {
+      // Create the project associations
+      const projectSlugToIdMap = await this.getProjectSlugToIdMap(
+        dto.projectEnvironments
+          .map((pe) => pe.projectSlug)
+          .filter((slug) => slug)
```

Comment:
I'm not entirely sure what the `filter` construct does

Reply:
The cli implementation was causing undefined slug values.